BtSendData/app/src/main/res/layout/activity_welcome_page.xml

```diff
@@ -67,6 +68,7 @@
 
 
 
+    //if we set android:background="@drawable/background" what is the difference of using this ? looks like a lot of code :)
```

Comment:
whats the difference between doing this imageview and just referencing a .png as a background ? both seem to work and the latter is alot less code

Reply:
If you already tried and it worked perfectly then do it. I tried to reference it as a background and it didn't work for me.

Reply:
awesome , thanks :)